app/assets/stylesheets/partials/_modal.scss

```diff
@@ -0,0 +1,84 @@
+#modalOverlay {
```

Comment:
Avoid using id selectors<br>Rule set contains (11/10) properties<br>Selector `modalOverlay` should be written in lowercase with hyphens